config/env/dev.json

```diff
@@ -5,6 +5,7 @@
   "anchorControllerEnabled": "@@ANCHOR_CONTROLLER_ENABLED",
   "anchorLauncherUrl": "@@ANCHOR_LAUNCHER_URL",
   "expirationPeriod": "@@ANCHOR_EXPIRATION_PERIOD",
+  "includeBlockInfoInAnchorProof": "@@INCLUDE_BLOCK_INFO_IN_ANCHOR_PROOF",
```

Comment:
is there a clay.json we need to update also?

Reply:
Good question, there are 3 files: `dev.json`, `prod.json`, and `test.json`. No separate file for Clay.

Reply:
Just also checked that previously added config values went into `dev.json` and `prod.json` and not in other places.